redis/client.py

```diff
@@ -203,6 +206,13 @@ def __init__(
         redis_connect_func=None,
         credential_provider: Optional[CredentialProvider] = None,
         protocol: Optional[int] = 2,
+        client_caching: bool = False,
+        client_cache: Optional[_Cache] = None,
```

Comment:
Can we pull the default string values (lru) into the cache.py also? DEFAULT_EVICTION_POLICY